contribute-documentation.md

```diff
@@ -39,6 +39,14 @@ The script compiles the Markdown files into static HTML pages and starts a local
 
 Please feel free to ask any questions you have on the developer mailing list.
 
+## Chinese documentation translation
+
+The Flink community is maintaining both English and Chinese documentation. If you want to update or extend the documentation, both English and Chinese documentation should be updated. If you are not familiar with Chinese language, you can open a JIRA with the `chinese-translation` component for Chinese documentation translation and link it with the current JIRA issue. If you are familiar with Chinese language, you are encouraged to update both sides in one pull request.
```

Comment:
"... you can open a JIRA with the `chinese-translation` component ..." 
-> 
"... please open a JIRA tagged with the `chinese-translation` component ..."